lib/api/webSockets.js

```diff
@@ -70,6 +70,10 @@ function handleResponse(msg) {
 			}); // resolve chain requests
 		} else if (res.result === 'success') {
 			req.resolve(res);
+		} else if (res.error && res.error.match(/device/igm)) {
+			// res.error can be Device is disabled, No device is available for test running ...
+			// some problem with device reject with appropriate msg
+			req.reject(new SuitestError(texts['ws.err.deviceProblem'](res.error), SuitestError.WS_ERROR));
 		} else {
 			req.reject(new SuitestError(texts.wsNotConnected() + (res.error ? ` ${res.error}.` : ''), SuitestError.WS_ERROR));
```

Comment:
I think this line is wrong in general. We are throwing `WebSocket connection with Suitest server is not established` on any server response with error. Shouldn't we only display `texts.wsNotConnected()` on `ws.error`? And for any server response with error we could just use message from `content.error`.